### PR TITLE
Feature/KAS-4726 handle refused parliament flows

### PR DESCRIPTION
**Pull request by tdn**
TBD if the status "niet ingeschreven" works at the VP side.

Also mind the `app-kaleidos` and `frontend-kaleidos` PRs to visualize the status

### CODE REVIEW

File: lib/sync.js [hunk 1]
```diff
@@ -102,12 +115,22 @@ async function syncSubmittedFlows(isSudo) {
...
       isSudo
     );
+    const refusedFlows = docs
+      .filter((doc) => doc.status === VP_PARLIAMENT_FLOW_STATUSES.REFUSED)
+      .map(doc => { return { parliamentId: `${doc.pobj}` } });
+    await updateParliamentFlowStatus(
+      refusedFlows,
+      PARLIAMENT_FLOW_STATUSES.REFUSED,
+      isSudo
+    );
```
Comment (ValenberghsSven) [inline lib/sync.js]: Are those the only 2 statuses we are going to get from that API call?
Could we just filter out all REFUSED and handle those separately from the "not" REFUSED?

**Review by ValenberghsSven — APPROVED** [review]
still need to test manually (when we can merge and test more of these VP pr's)

**Comment by ValenberghsSven** [comment]
need to discuss this with VP. Maybe they would rather have a separate endpoint for refused flows

File: lib/sync.js (hunk 1 above)
Comment (tdn) [reply lib/sync.js]: No, there are multiple statuses possible

**Comment by tdn** [comment]
> need to discuss this with VP. Maybe they would rather have a separate endpoint for refused flows

Discussed it today. Normally this code should work, but will wait until they have it on their side to test and merge

**Comment by tdn** [comment]
tested successfully today: 
image

**Pull request merged**